packages/next/src/build/webpack/loaders/next-flight-action-entry-loader.ts

```diff
@@ -20,18 +20,13 @@ function nextFlightActionEntryLoader(this: any) {
     .flat()
 
   return `
-${individualActions
-  .map(([_, path]) => {
-    // This import ensures that the module is always bundled even if there's no
-    // explicit import in the codebase, to avoid the action being DCE'd.
-    return `import(/* webpackMode: "eager" */ ${JSON.stringify(path)});`
-  })
-  .join('\n')}
-
 ${individualActions
   .map(([id, path, name]) => {
     // Re-export the same functions from the original module path as action IDs.
-    return `export { ${name} as "${id}" } from ${JSON.stringify(path)}`
+    return `
+    const { ${name}: _${id} } = await import(${JSON.stringify(path)});
```

Comment:
This creates the following warning during build:
```
The generated code contains 'async/await' because this module is using "topLevelAwait".
However, your target environment does not appear to support 'async/await'.
As a result, the code may not run as expected or may cause runtime errors.
```

I didn't notice an issue though. _But_, we actually don't need the dynamic import, it also works with normal imports. But then we probably need to find a way to set the `async` flag in the server module map dynamically during build time based on the module meta data.